corehq/apps/locations/models.py

```diff
@@ -77,6 +77,7 @@ class LocationType(models.Model):
     administrative = models.BooleanField(default=False)
     shares_cases = models.BooleanField(default=False)
     view_descendants = models.BooleanField(default=False)
+    last_modified = models.DateTimeField(auto_now=True)
```

Comment:
I actually can't say from my own experience because I just haven't used this before; but @czue's comments make sense and I have seen this used elsewhere like he said. My read is that this won't cause any timezone bugs, but only because our django project timezone (what a bizarre concept) is UTC.